src/expressions/operators/arithmetic/Unary.ts

```diff
@@ -6,12 +6,7 @@ import sequenceFactory from '../../dataTypes/sequenceFactory';
 import { SequenceType, ValueType } from '../../dataTypes/Value';
 import Expression from '../../Expression';
 
-type UnaryLookupTable = {
-	[key: number]: ValueType;
-};
-
-// TODO: fix this?
-const UNARY_LOOKUP: UnaryLookupTable = {
+const UNARY_LOOKUP: { [key: number]: ValueType } = {
```

Comment:
That looks a lot cleaner indeed!

Reply:
Emm duplicated again, it's being done in the review changes branch which is still on going. But not a problem by itself.